### PR TITLE
feat: Adds a resolved path for output

### PR DESCRIPTION
**Pull request by pksjce**
**What kind of change does this PR introduce?**
This tries to address #73 

**Did you add tests for your changes?**
Yep

**If relevant, did you update the documentation?**
NA

**Summary**
To summarise, it tries to implement what was promised in #73

### CODE REVIEW

File: lib/transformations/utils.test.js [hunk 1]
```diff
@@ -157,4 +157,11 @@ var a = { plugs: [] }
 			.filter(p => utils.findObjWithOneOfKeys(p, ['a'])).size()).toEqual(1);
 		});
 	});
+
+	describe('getRequire', () => {
+		it('should create a require statement', () => {
+			const require = utils.getRequire(j, 'filesys', 'fs');
+			expect(require).toMatchSnapshot();
```
Comment (okonet) [inline lib/transformations/utils.test.js]: This should probably create snapshot for the source code. Right now it's for AST.

`expect(require.toSource()).toMatchSnapshot()`?

File: lib/transformations/outputPath/__testfixtures__/outputPath-1.input.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const path = require('path');
+module.exports = {
+	output: {
+		path: path.join(__dirname, 'dist')
```
Comment (okonet) [inline lib/transformations/outputPath/__testfixtures__/outputPath-1.input.js]: Should we add a test where `path` is imported as a variable with a different name?

`const p = require('path')` should also work I guess.

Comment (pksjce) [inline lib/transformations/outputPath/__testfixtures__/outputPath-1.input.js]: Oh yeah, what is path is required in the file but `output` doesnnt use it? Need to add some more checks.

File: lib/transformations/utils.test.js (hunk 1 above)
Comment (pksjce) [inline lib/transformations/utils.test.js]: Fragments dont actually have `.toSource()` on them. :( Will try to figure out if this is possible some other way. I think I should be able to convert it to  a node somehow.

File: lib/transformations/outputPath/__testfixtures__/outputPath-1.input.js (hunk 2 above)
Comment (pksjce) [inline lib/transformations/outputPath/__testfixtures__/outputPath-1.input.js]: Added this.

File: lib/transformations/utils.test.js (hunk 1 above)
Comment (pksjce) [inline lib/transformations/utils.test.js]: Had to wrap it in a `j()`

File: lib/transformations/outputPath/outputPath.js [hunk 3]
```diff
@@ -6,12 +6,22 @@ module.exports = function(j, ast) {
...
 			.find(j.VariableDeclarator)
-			.filter(p => utils.safeTraverse(p, ['value', 'id', 'name']) === 'path').size();
+			.filter(p => utils.safeTraverse(p, ['value', 'init', 'callee', 'name']) === 'require')
+
+			.filter(p => utils.safeTraverse(p, ['value', 'init', 'arguments']) && p.value.init.arguments.reduce((isPresent, a) => {
+				return a.type === 'Literal' && a.value === 'path' || isPresent;
+			}, false));
+		if (pathDecalaration) {
+			isPathPresent = true;
+			pathDecalaration.forEach(p => {pathVarName = utils.safeTraverse(p, ['value', 'id', 'name']);});
```
Comment (okonet) [inline lib/transformations/outputPath/outputPath.js]: Let's put the block code on a new line please. This is hard to read now.

File: lib/transformations/outputPath/outputPath.js [hunk 4]
```diff
@@ -6,12 +6,22 @@ module.exports = function(j, ast) {
...
+		}
+		console.log(pathVarName);
```
Comment (okonet) [inline lib/transformations/outputPath/outputPath.js]: Let's remove `console.log` calls please.

File: lib/transformations/outputPath/outputPath.js [hunk 5]
```diff
@@ -6,12 +6,22 @@ module.exports = function(j, ast) {
...
+		literalOutputPath.find(j.Literal)
```
Comment (okonet) [inline lib/transformations/outputPath/outputPath.js]: Should the `.find()` be also a new line?

**Comment by pksjce** [comment]
Hey @okonet - Thanks for the detailed review. I've taken care of the changes.

**Comment by okonet** [comment]
Feel free to squash & merge with the commitizen-compatible commit message!

**Pull request merged**